Source/Bogus.Tests/GitHubIssues/Issue124.cs

```diff
@@ -129,24 +129,34 @@ public void nullable_int_and_nullable_reference_type()
             .RuleFor(x => x.Obj, f => new object().OrNull(f))
             .RuleFor(x => x.Str, f => f.Random.Word().OrNull(f));
 
-         var q = faker.Generate(3);
+         var q = faker.Generate(5);
```

Comment:
Increased to 5 to ensure that at least one of the objects has a non-null `Str` value.